profiles/ug/modules/ug/ug_system/ug_system.module

```diff
@@ -1,8 +1,10 @@
 <?php
+// DEFINE('API_URL', "https://app.lumturio.com/api/");
+// DEFINE('API_TOKEN', "7tMeJBFO3EYx5K2iYsgNzpBiI84Jws");
```

Comment:
I don't think we can include our API key in the repository. You will need to accept the API key via a field on the Lumturio configuration form. The API_URL can be included in the code, but I would make it a variable with a default value rather than a defined constant.